javascript/problem1.js

```diff
@@ -0,0 +1,8 @@
+function myFunc(string){
+   
+
+   let msg=  (string.length <= 3) ? 'String is too short' : string.slice(-3)+string+string.slice(-3);
```

Comment:
The variable name msg could be renamed to something more meaningful like result or outputMessage for better readability

Reply:
The function currently checks for strings of length 3 or less and returns 'String is too short'. However, it's better to be consistent with the string length you are working with
ex=> the string should be longer than 3 characters